package.json

```diff
@@ -189,6 +189,7 @@
     "stylis-plugin-rtl-sc": "npm:stylis-plugin-rtl@^1.1.0",
     "terser-webpack-plugin": "^5.3.6",
     "thenify": "3.3.1",
+    "ts-node": "^10.9.1",
```

Comment:
babel does not work?

Reply:
Babel does not respect tsconfig.json. With ts-node I'm following the pattern used in Toolpad